src/app/wallet/wallet/shared/transaction-table/transaction-table.component.html

```diff
@@ -28,7 +28,7 @@
         <mat-panel-title fxFlex="1 0 50px" fxFlex.lt-md="100" *ngIf="display.confirmations" class="history_confirmation">
```

Comment:
Could you please add the dynamic classes to this element instead?

Reply:
aah sure

Reply:
@allienworks done please check now